godot-macros/src/class/data_models/field_var.rs

```diff
@@ -200,10 +201,14 @@ impl GetterSetterImpl {
             }
         }
 
+        let constant_declaration =
```

Comment:
```suggestion
        let func_name_constant =
```
To make clear that it's not a user-defined `#[constant]`.